homeassistant/components/vacuum/dyson.py

```diff
@@ -7,14 +7,14 @@
 import asyncio
 import logging
 
+from homeassistant.util.icon import icon_for_battery_level
```

Comment:
No, please put this back below. Imports should be alphabetically sorted but following the greater rules of PEP8 for imports.